include/spdk/nvmf.h

```diff
@@ -83,6 +83,7 @@ struct spdk_nvmf_transport_opts {
 	bool		no_srq;
 	bool		c2h_success;
 	bool		dif_insert_or_strip;
+	bool            wr_batching;
```

Comment:
Interface changes must be reflected in CHANGELOG.md.
Changes in RPC and config file should also be added to changelog in their corresponding commits.